src/APIBasedAuth.ts

```diff
@@ -87,7 +85,8 @@ const DEFAULT_STORAGE_KEYS: APIBasedAuth.StorageKeys = {
 
 class APIBasedAuth {
   private client: AxiosInstance;
-  private clientOptions: APIBasedAuth.Config;
+  clientOptions: APIBasedAuth.Config;
```

Comment:
exposing this for testing purposes for clients to ensure they have configured options properly or to access anything they need after the fact

Reply:
Should we mark this as `readonly`?